src/ansiblelint/rules/no_handler.py

```diff
@@ -69,7 +69,11 @@ def matchtask(
         task: Task,
         file: Lintable | None = None,
     ) -> bool | str:
-        if task["__ansible_action_type__"] != "task" or task.is_handler():
```

Comment:
Was not possible to fix `is_hander()` to ensure that all tasks from a handlers tasks files are reported as true?

IMHO, that would make it much easier to maintain, and probably it would also allow us to simplify the code.

Reply:
@ssbarnea Updated. Now the check is in the `is_handler` function